src/auth/default/DefaultAuthProvider.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import DefaultAuthContext from 'auth/default/DefaultAuthContext';
-import { history, ROUTE_HOME } from 'app-init/router';
+import { history, ROUTE_HOME, ROUTE_DASHBOARD } from 'app-init/router';
 
 const DefaultAuthProvider = ({ children }) => {
   const auth = {
     enabled: false,
     authenticated: false,
-    logout: () => history.push(ROUTE_HOME),
+    logout: (status) => {
+      let addredirect = '';
+      if (status) {
```

Comment:
I'd suggest to use const and binary operator (extracting some logic to a new function to ease the process)

Reply:
done